### PR TITLE
Use "Checking" instead of "Current" for US banks

### PR DESCRIPTION
**Pull request by AlBillington**
I noticed the bank info page used the term "Current Accounts" which I hadn't ever heard of but learned is common in the UK.  Added logic to use "checking account" for US banks

### CODE REVIEW

File: components/eco-bank/EcoBankDetail.vue [hunk 1]
```diff
@@ -52,7 +52,7 @@
       <template #products>
         <div class="grid grid-cols-1 md:grid-cols-3 gap-4 md:-mr-24">
           <p>
-            <strong>Current Accounts:</strong>
+            <strong>{{ isBE() ? 'Checking' : 'Current' }} Accounts:</strong>
```
Comment (claudiabdm) [inline components/eco-bank/EcoBankDetail.vue:55]: I think this should be the other way around, right? 
```suggestion
            <strong>{{ isBE() ? 'Current' : 'Checking' }} Accounts:</strong>
```

File: components/eco-bank/EcoBankDetail.vue [hunk 2]
```diff
@@ -248,6 +250,7 @@ const props = defineProps<{
   website: string;
   rating: string;
   bankFeatures: BankFeature[];
+  countries: { code: string; }[];
```
Comment (claudiabdm) [inline components/eco-bank/EcoBankDetail.vue:253]: Why is this prop added? I don't see it being used anywhere in this PR 😅

**Review by claudiabdm — CHANGES_REQUESTED** [review]

File: components/eco-bank/EcoBankDetail.vue (hunk 2 above)
Comment (AlBillington) [reply components/eco-bank/EcoBankDetail.vue]: ah I originally was using it but then found the helper, will remove

**Review by claudiabdm — APPROVED** [review]
Looks good now! Feel free to merge.

**Pull request merged**